resources/proto/message-payload.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package com.ziggurat.proto;
+
+option java_package = "com.ziggurat.proto";
+option java_outer_classname = "MessagePayloadProto";
+
+message MessagePayload {
+  bytes  msg                = 1;
+  string topic_entity_name  = 2;
```

Comment:
Are we sticking with the name `topic_entity`?

Reply:
done